### ISSUE TITLE
Change "lambda" to "lambda_soi" for python compatibility reasons and naming convention

### ISSUE THREAD

**Issue by pfannkuch3n** [RWTH-EBC/AixLib#1538]
**What is the problem?**

In DHC-pipes models AixLib.Fluid.DistrictHeatingCooling.Pipes.PlugFlowPipeEmbedded and DHCPipe
Heat conductivity of material/soil is named "lambda" while other soil-related parameters ending with _soi. When writing modelica code with python "lambda" requires a workaround, since its a reserved keyword in python.

**How do we want to solve it? Describe the solution you'd like**

I would suggest change lambda to lambda_soi (would be, in my opinion, more consistent with remaining variables)

**Comment by FWuellhorst** [comment]
Good idea. Consider sticking to our camel case convention `lambdaSoi`.
